armory/scenarios/base.py

```diff
@@ -311,12 +322,16 @@ def init_interactive(config_json, from_file=True):
             "--num_eval_batches will be overwritten and set to 1 since --check was passed"
         )
 
+    if args.skip_benign and args.skip_attack:
+        raise ValueError("--skip-benign and --skip-attack cannot both be passed")
+
```

Comment:
Ditto on this. When I comment this and the above similar lines in `__main__`, it runs smoothly and has `{}` for results, which is expected.